benchopt/plotting/generate_html.py

```diff
@@ -181,21 +181,26 @@ def shape_solvers_for_html(df, objective_column):
         df_filtered = df_filtered.dropna(subset=[objective_column])
 
         q1, q9 = compute_quantiles(df_filtered)
+        groupby_stop_val_median = df_filtered.groupby('stop_val').median()
         color, marker = get_solver_style(solver)
+
+        stopping_strategy = df_filtered['stopping_strategy'].unique()
```

Comment:
Shall we consider a sanity check here: if run, a solver can have only one stopping strategy?

Reply:
You can always add the sanity check, it is better to fail fast than get weird behaviors :)